test/jest/aliasUtils.spec.ts

```diff
@@ -1,20 +1,27 @@
-import { getAlias, isAlias } from '../../lib/utils/aliasUtils';
+import { getAlias, isNodePubKey } from '../../lib/utils/aliasUtils';
+
+const nodePubKey = '028599d05b18c0c3f8028915a17d603416f7276c822b6b2d20e71a3502bd0f9e0a';
+const invalidPubKey = 'notarealnodepubkey028915a17d603416f7276c822b6b2d20e71a3502bd0f9e0a';
+const alias = 'SoccerFrost';
 
 describe('getAlias', () => {
-  test('generates a deterministic alias from public key', () => {
-    const pubkey = '028599d05b18c0c3f8028915a17d603416f7276c822b6b2d20e71a3502bd0f9e0a';
-    const alias = 'SoccerFrost';
-    expect(getAlias(pubkey)).toEqual(alias);
+  test('generates a deterministic alias from a node pub key', () => {
+    expect(getAlias(nodePubKey)).toEqual(alias);
+  });
+
+  test('throws an error when passed an node pub key', () => {
```

Comment:
```suggestion
  test('throws an error when passed a node pub key', () => {
```

Reply:
This was actually supposed to say "an invalid node pub key" but I guess I forgot to type out "invalid"